src/Http/src/Request/InputManager.php

```diff
@@ -92,8 +107,11 @@ final class InputManager implements SingletonInterface
 
     public function __construct(
         /** @invisible */
-        private ContainerInterface $container
+        private readonly ContainerInterface $container,
+        /** @invisible */
+        HttpConfig $config = new HttpConfig()
     ) {
+        $this->bagAssociations = \array_merge($this->bagAssociations, $config->getInputBags());
```

Comment:
Maybe we will get all aliases and create a new array with `alias` => `name`?

Reply:
In this case, need to add a new method addInputBagAlias so that allow can add an alias. Now it can be done like this:
https://github.com/spiral-packages/symfony-validator/blob/master/src/Bootloader/ValidatorBootloader.php#L31